### ISSUE TITLE
Overlay vs Routed Network Type in 1.3.0

### ISSUE THREAD

**Issue by paalth** [apprenda/kismatic#527]
Used the following networking values in kismatic-cluster.yaml:

```
  networking:
    type: routed
    pod_cidr_block: 10.233.0.0/16
    service_cidr_block: 10.232.0.0/16
    policy_enabled: false
    update_hosts_files: true
```

It does not appear that the networking type of 'routed' was used as the nodes created end up configured with IP-in-IP tunnels: 

```
[kismaticuser@k8smaster0 ~]$ route
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
default         10.231.0.1      0.0.0.0         UG    100    0        0 eth0
10.231.0.0      0.0.0.0         255.255.0.0     U     100    0        0 eth0
10.233.88.0     k8singress0.nov 255.255.255.192 UG    0      0        0 tunl0
10.233.119.192  0.0.0.0         255.255.255.192 U     0      0        0 *
10.233.159.0    k8sworker2.nova 255.255.255.192 UG    0      0        0 tunl0
10.233.195.0    k8sworker1.nova 255.255.255.192 UG    0      0        0 tunl0
10.233.233.64   k8sworker0.nova 255.255.255.192 UG    0      0        0 tunl0
169.254.169.254 10.231.0.1      255.255.255.255 UGH   100    0        0 eth0
172.17.0.0      0.0.0.0         255.255.0.0     U     0      0        0 docker0
[kismaticuser@k8smaster0 ~]$

```
Did not encounter this issue previously when using v1.2.0 

The environment used is RHEL 7.2 on OpenStack VMs.

**Comment by alexbrand** [comment]
Thanks @paalth 
To clarify, you have been using kismatic with `routed` networking instead of `overlay` in the past?

**Comment by alexbrand** [comment]
Hey @paalth, would it be possible for you to test the change that is in #529?

**Comment by paalth** [comment]
The change that is in #529 has missing quotes around the values (the values are strings). 

After adding quotes I ran another manual test and routed mode seems to be correctly configured:

```
[kismaticuser@k8smaster0 ~]$ route
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
default         10.231.0.1      0.0.0.0         UG    100    0        0 eth0
10.231.0.0      0.0.0.0         255.255.0.0     U     100    0        0 eth0
10.233.159.0    k8sworker2.nova 255.255.255.192 UG    0      0        0 eth0
10.233.233.64   k8sworker0.nova 255.255.255.192 UG    0      0        0 eth0
169.254.169.254 10.231.0.1      255.255.255.255 UGH   100    0        0 eth0
172.17.0.0      0.0.0.0         255.255.0.0     U     0      0        0 docker0
[kismaticuser@k8smaster0 ~]$

```
The installation doesn't complete successfully yet (the DNS pods keep crashing due to some network config problem), but I believe the issue with the installer not using the configured network type could be consider fixed if the quotes are added in #529 as mentioned above.
